app/kubemci/pkg/validations/validations.go

```diff
@@ -92,6 +94,7 @@ func nodePortSameInAllClusters(backend v1beta1.IngressBackend, namespace string,
 	return nil
 }
 
+// ServerVersionsNewEnough returns an error if the cluster Kubernetes server versions are not supported.
```

Comment:
"cluster Kubernetes server versions" -> "cluster version is"

Reply:
Done, and reworded a little.